ch05construction/09patterns.ipynb

```diff
@@ -380,112 +420,94 @@
     "        self.end = self.data[-1, 0]\n",
     "        self.range = self.end - self.start\n",
     "        self.step = self.range / self.count\n",
-    "        self.times = self.data[:, 0]\n",
-    "        self.values = self.data[:, 1]\n",
+    "        self.times = self.data[:, 0].copy()\n",
+    "        self.values = self.data[:, 1].copy()\n",
```

Comment:
Do we want to point out why we're copying, maybe in a comment? (I'm missing the context right now, may be obvious)

Reply:
👍 lomb scargle now has to take in an array,  not a view of an array